packages/eslint-config-jest/README.md

```diff
@@ -0,0 +1,5 @@
+# `@bedrockstreaming/eslint-config-jest`
```

Comment:
Maybe you could append "Install" & "Usage" chapters by getting the same structure that you can find in other Readme files